packages/frontend/src/ee/pages/AiAgents/AgentsWelcome.tsx

```diff
@@ -58,23 +57,18 @@ const AGENT_FEATURES = [
 
 const AgentsWelcome = () => {
     const { projectUuid } = useParams();
-    const { data: agents, isLoading } = useProjectAiAgents(projectUuid);
     const navigate = useNavigate();
     const canCreateAgent = useAiAgentPermission({ action: 'manage' });
+    const [defaultAgentUuid] = useDefaultAgent(projectUuid);
 
     useEffect(() => {
-        if (agents && agents.length > 0) {
-            // TODO: Get this from user preferences (tbi) or prev. used from local storage
-            const firstAgent = agents[0];
+        if (defaultAgentUuid) {
             void navigate(
-                `/projects/${projectUuid}/ai-agents/${firstAgent.uuid}`,
+                `/projects/${projectUuid}/ai-agents/${defaultAgentUuid}`,
             );
         }
-    }, [agents, navigate, projectUuid]);
+    }, [defaultAgentUuid, navigate, projectUuid]);
```

Comment:
this `useEffect` can be substituted with [`Navigate`](https://reactrouter.com/api/components/Navigate#navigate) component

Reply:
Updated 🙏